### DISCUSSION TITLE
NRH removed from Chrome Play store

### DISCUSSION THREAD

**Discussion by AtomicRobotMan0101** [sr33/ares#27] [General]
Anyone know why?

**Comment by ghost** [comment]
> Anyone know why?

I have the same question too. I also created a reddit thread to get answers, you can check it here and see if someone figures this out:

https://www.reddit.com/r/antivirus/comments/v4wze8/nuke_reddit_history_is_malware/?utm_medium=android_app&utm_source=share

**Comment by mondocane80** [comment]
Looks like this is abandonware.  The Firefox extension has been removed as well.
